frameworks/hdfs/universe/config.json

```diff
@@ -60,6 +60,30 @@
 					"type": "string",
 					"description": "The placement strategy for HDFS nodes. This should always be NODE.",
 					"default": "NODE"
+				},
+				"secure_mode": {
+					"id": "http://hdfs/dcos/mesosphere.com/service/secure_mode",
+					"type": "boolean",
+					"description": "Enable HDFS Secure Mode",
+					"default": false
+				},
+				"keytabs_uri": {
+					"id": "http://hdfs/dcos/mesosphere.com/service/keytabs_uri",
+					"type": "string",
+					"description": "The URI from which the Keytabs are downloaded.",
+					"default": "http://10.0.1.69:8788/keytabs.tar.gz"
+				},
+				"task_jre_uri": {
+					"id": "http://hdfs/dcos/mesosphere.com/service/task_jre_uri",
+					"type": "string",
+					"description": "The URI from which the JRE for all tasks is downloaded",
+					"default": "https://s3-us-west-2.amazonaws.com/infinity-artifacts/jre-8u91-linux-x64.tar.gz"
+				},
+				"user": {
+					"id": "http://hdfs/dcos/mesosphere.com/service/user",
+					"type": "string",
+					"description": "The user as which service and tasks are run.",
```

Comment:
This is odd wording.  Let's not be afraid of preposition-at-the-end-of-sentence: "The user the service and tasks are run as."